.github/workflows/trivy-containers.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     
     steps:
       - name: Scan container image
-        uses: aquasecurity/trivy-action@master
```

Comment:
To clarify: We're hardcoding the version here to not be caught off guard by a breaking change?